src/style/color-palette-extended-light-mode-only.css

```
@@ -7,14 +7,24 @@
  */
 
 :root {
-    /* Lime Technologies Brand Colors (Do not have dark/light mode variants) */
+    /* Lime Technologies Brand Colors */
+    --lime-brand-color-lime-green: 190, 224, 52; /* #BEE034 */
+    --lime-brand-color-ocean-teal: 111, 205, 182; /* #6FCDB6 */
+    --lime-brand-color-aqua: 	166, 239, 255; /* #A6EFFF */
+    --lime-brand-color-bubble-gum: 	255, 166, 234; /* #FFA6EA */
+    --lime-brand-color-sunny-orange: 254, 176, 0; /* FEB000 */
```

Consistency nitpick 😄

```suggestion
    --lime-brand-color-sunny-orange: 254, 176, 0; /* #FEB000 */
```

`fixup! feat(style): add CSS variables for new Lime brand colors`